packages/database/src/migrations/20201103092344-AddPSSSProject-modifies-data.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PROJECT_CODE = 'psss';
+const DASHBOARDGROUP_NAME = 'PSSS';
+const USER_GROUP = 'PSSS';
+const NZ_COUNTRY_CODE = 'NZ';
+
+const COUNTRY_LIST = [
+  'AS',
+  'CK',
+  'FM',
+  'FJ',
+  'PF',
+  'GU',
+  'KI',
+  'MH',
+  'NR',
+  'NC',
+  'NU',
+  'NZ', // New Zealand needs to be added to db
+  'MP',
+  'PW',
+  'PG',
+  'PI',
+  'WS',
+  'SB',
+  'TK',
+  'TO',
+  'TV',
+  'VU',
+  'WF',
+];
+
+export const hierarchyNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM entity_hierarchy WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+exports.up = async function (db) {
+  // Add country + entity for New Zealand
+  await insertObject(db, 'entity', {
+    id: generateId(),
+    code: NZ_COUNTRY_CODE,
+    country_code: NZ_COUNTRY_CODE,
+    parent_id: await codeToId(db, 'entity', 'World'),
+    name: 'New Zealand',
+    type: 'country',
+  });
+  await insertObject(db, 'country', {
+    id: generateId(),
+    code: NZ_COUNTRY_CODE,
+    name: 'New Zealand',
+  });
+
+  // Add project entity and entity hierarchy for PSSS
+  await insertObject(db, 'entity', {
+    id: generateId(),
+    code: PROJECT_CODE,
+    parent_id: await codeToId(db, 'entity', 'World'),
+    name: DASHBOARDGROUP_NAME,
```

Comment:
Kinda weird variable name to use here. It is the same though, so maybe just a variable rename?